docs/source/examples/spot-jobs.rst

```diff
@@ -241,3 +241,34 @@ you can still tear it down manually with
 
 .. note::
   Tearing down the spot controller will lose all logs and status information for the spot jobs and can cause resource leakage when there are still in-progress spot jobs.
+
+Customize spot controller resources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Although the default setting works for most of people, you may want to customize the resources of the spot controller for the following reasons:
+
+1. Enforcing the spot controller to run on a specific location. (Default: cheapest location)
+2. Changing the maximum number of spot jobs that can be run concurrently. (Default: 16)
+3. Changing the disk_size of the spot controller to store more logs. (Default: 50GB)
+4. Using a specific instance type for the spot controller. (Default: choose automatically)
+
+To achieve the above goals, you can specify the configs in the :code:`~/.sky/skypilot_config.yaml` with the following fields (the :code:`resources` field has the same spec as a normal SkyPilot job, see `here <https://skypilot.readthedocs.io/en/latest/reference/yaml-spec.html>`_):
+
+.. code-block:: yaml
+
+    spot:
+      controller:
+        resources:
+          # All the configs below are optional
+          # 1. Specify the location of the spot controller.
+          cloud: gcp
+          region: us-central1
+          # 2. Specify the maximum number of spot jobs that can be run concurrently.
+          cpus: 4+ # number of vCPUs, max # spot jobs = 2 * cpus
+          # 3. Specify the disk_size of the spot controller.
+          disk_size: 100
+          # 4. Specify the instance type of the spot controller.
+          instance_type: n1-standard-4
```

Comment:
I feel like it's ok to leave out this field here and the bullet point 4 above, to make it slightly more portable. We already have `cpus`. Wdyt?